sentinelsat/sentinel.py

```diff
@@ -939,24 +885,51 @@ def check_bounds(geometry):
     wkt = re.sub(r'(?<!\d) ', '', wkt)
     return wkt
 
-
+def placename_to_wkt(placename):
+    """Maps the placename to rectangular bounding extents using Nominatim API and converts to Well-Known-Text.
+    Parameters
```

Comment:
Include whitespace to be consistent with the other function definitions in the code.